README.md

```diff
@@ -1,49 +1,48 @@
-| -------------------------------------------- | ------------------------- | -------------------------- | -------------------------------------------------------------------- |
-| ClearBlade System Key                                   | `cbSystemKey`               | N/A            | `Yes`                                                                  |
-| ClearBlade User Token                            | `cbToken`             | N/A                        | `Yes`                                              |
-| ClearBlade Registry Region                            | `cbRegistryRegion`             | N/A                        | `No`                                              |
-| Google IoT Core Registry Name                  | `gcpRegistryName`            | N/A                        | `Yes`                                                     |
-| Google IoT Core Registry Region                | `gcpRegistryRegion`              | N/A                        | `Yes` |
-| GCP Service account file path               | `gcpServiceAccount`              | N/A                        | `Yes` |
-| Device to migrate CSV file path  | `devicesCsv`                | N/A                        | `No`                                                                  |
-| Connect to IoT Core sandbox system                        | `sandbox`                      | `false`       | `No`                                                                  |
-| Update public keys for existing devices                 | `updatePublicKeys`                       | `true` | `No`                                                                  |
-| Store Config Version History                 | `configHistory`                       | `false` | `No`                                                                  |
-
-  
+| Name                                                                                                                                                                   | CLI Flag            | Default                  | Required |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | ------------------------ | -------- |
+| Path to ClearBlade Service Account File ([see here for more info](https://clearblade.atlassian.net/wiki/spaces/IC/pages/2240675843/Add+service+accounts+to+a+project)) | `cbServiceAccount`  | N/A                      | `Yes`    |
+| ClearBlade Registry Region                                                                                                                                             | `cbRegistryName`    | N/A                      | `Yes`    |
+| ClearBlade Registry Region                                                                                                                                             | `cbRegistryRegion`  | `region of GCP registry` | `No`     |
```

Comment:
the flag is `cbRegistryRegion` but the description is `region of GCP registry`. Should be the region of the ClearBlade registry.

Reply:
Oh, I think I misread that. The default is whatever the region of GCP registry is right?